node-js-basics-101/node_modules/socket.io/lib/index.js

```diff
@@ -0,0 +1,474 @@
+
+/**
+ * Module dependencies.
+ */
+
+var http = require('http');
+var read = require('fs').readFileSync;
+var path = require('path');
+var exists = require('fs').existsSync;
+var engine = require('engine.io');
+var clientVersion = require('socket.io-client/package.json').version;
+var Client = require('./client');
+var Emitter = require('events').EventEmitter;
+var Namespace = require('./namespace');
+var Adapter = require('socket.io-adapter');
+var parser = require('socket.io-parser');
+var debug = require('debug')('socket.io:server');
+var url = require('url');
+
+/**
+ * Module exports.
+ */
+
+module.exports = Server;
+
+/**
+ * Socket.IO client source.
+ */
+
+var clientSource = undefined;
+var clientSourceMap = undefined;
```

Comment:
__[jshint]__
* It's not necessary to initialize 'clientSourceMap' to 'undefined'.

<!-- Comment by [SideCI](https://sideci.com) -->